protobufs/livekit_room.proto

```diff
@@ -228,3 +233,17 @@ message RoomConfiguration {
   // Define agents that should be dispatched to this room
   repeated RoomAgentDispatch agents = 10;
 }
+
+message ForwardParticipantRequest {
+  // room to forward participant from
+  string room = 1;
+  // identity of the participant to forward
+  string identity = 2;
+  // room to forward participant to
+  string destination_room = 3;
```

Comment:
Should this be a list if a participant can be forwarded to many rooms?

Reply:
it can be called multiple times to forward to many rooms and I think it is better than a list for reasons:

Reply:
it can be called multiple times to do that. it would be better than a list since it is simple to process error or retry when the list version is partial succeed.

Reply:
Sounds good.